packages/subgraph/schema.graphql

```diff
@@ -462,152 +452,356 @@ type StandardCallback @entity {
   permissionId: Bytes!
+
+  " The address (EOA or contract) to which the permission has been granted. "
   who: Bytes!
+
+  " The address of the context in which the permission has been granted. "
   actor: Bytes!
+
+  " The address of the contract containing the conditions which authorize a call by another address. This contract must inherit from `PermissionCondition` and implement the `IPermissionCondition` interface. "
   condition: Bytes
 
-  # The following attributes is used to create
+  # The following attributes are used to create
   # one-to-many between DAO or PluginRepo to Permission
+
+  " The DAO that grants the permission (DAO uses `PermissionManager` SC `grant` function). "
   dao: Dao
+
+  " The PluginRepo that grants the permission (PluginRepo uses `PermissionManager` SC `grant` function). "
   pluginRepo: PluginRepo
 }
 
-# Plugins
+#############################
+######### Plugins ###########
+#############################
 
+"""
+PluginRepo are contracts themselves, handling the versioning of its associated unique plugin. They are deployed using the [PluginRepoFactory](https://devs.aragon.org/docs/osx/how-it-works/framework/plugin-management/plugin-repo/plugin-repo-creation/).
+"""
 type PluginRepo @entity(immutable: true) {
-  id: ID! # address
+  " The address of the PluginRepo contract. "
+  id: ID!
+
+  " Each PluginRepo has to have a valid ENS subdomain associated, added while creating and registering it. "
   subdomain: String!
+
+  " The different incompatible versions for the associated plugin.  "
   releases: [PluginRelease!]! @derivedFrom(field: "pluginRepo")
-  # Holds all preparations. Also applied ones.
+
+  " The prepared necessary settings and addresses to install/uninstall to a DAO the plugin associated to a `PluginRepo`. Also applied ones. "
   preparations: [PluginPreparation!]! @derivedFrom(field: "pluginRepo")
-  # Holds all installed and uninstalled installations.
+
+  # The plugins installed and uninstalled to DAOs associated to each `PluginRepo`.
   installations: [PluginInstallation!] @derivedFrom(field: "appliedPluginRepo")
 
+  " The permissions given to a PluginRepo to govern its actions (e.g. MAINTAINER_PERMISSION_ID to create a new plugin version). "
   permissions: [Permission!]! @derivedFrom(field: "pluginRepo")
 }
 
+"""
+The contracts that will deploy your plugin using different factory patterns (new/minimal proxy pattern/UUPS pattern).
+Pointing to a specific version of the plugin (the address for one specific implementation contract).
+"""
 type PluginSetup @entity(immutable: true) {
-  id: ID! # contract address
+  " The PluginSetup contract address. "
+  id: ID!
+
+  " The version of the plugin internally referenced in the PluginSetup. "
   versions: [PluginVersion!]! @derivedFrom(field: "pluginSetup")
 }
 
+"""
+Registration of the different incompatible versions of a plugin.
```

Comment:
Maybe say here that a release is dictated by a new version of a plugin being incompatible with an old version. Maybe also link to the Sem ver or releases section in the docs.